tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1493,6 +1493,126 @@ TEST_F(OnnxImporterTest, reduceMean2AvgPoolNoKeepDims) {
       checkConstFoldedOutput(netFilename, {"x"}, {&x}, {bindings.get(output)}));
 }
 
+/// Test loading ReduceMax op from a ONNX model.
+/// Input shape is 4D, two dimensions are reduced,Output shape is 4D.
+TEST_F(OnnxImporterTest, reduceMaxKeepDims) {
```

Comment:
This and the other tests are all essentially copies of the `reduceMin*` tests too right? Can we just create a helper here, e.g. see `testEltwiseUnaryOpFloat()`?

Reply:
Created `testReductionOps()` which tests `ReduceMin`, `ReduceMax` and `ReduceMean`.